### PR TITLE
Update syslog.php

### PR DESCRIPTION
**Pull request by Fireball2k15**
Found an error that does not alow hosts including a "." - so there was no way to translate unknown fqdns

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Hi @Fireball2k15 
Thanx for your PR. It seems that you have a few indent issues in your PR. Could you check the tests results (travis, here : https://travis-ci.com/librenms/librenms/jobs/271567703 ) and make the necessary corrections ? 
Thanx

File: includes/syslog.php [hunk 1]
```diff
@@ -57,9 +57,15 @@ function process_syslog($entry, $update)
     }
 
     $entry['host'] = preg_replace("/^::ffff:/", "", $entry['host']);
-    if ($new_host = Config::get("syslog_xlate.{$entry['host']}")) {
-        $entry['host'] = $new_host;
+    $axlate = Config::get('syslog_xlate');
+    if(true == is_array($axlate)){
```
Comment (Jellyfrog) [inline includes/syslog.php]: This entire thing could be replaced with something like
```Php
if (is_array($axlate) && isset($axlate[$entry['host']]) {
  $entry['host'] = $axlate[$entry['host']];
}
```
If im not missing anything.

**Comment by PipoCanaja** [comment]
I would say that the issue is not linked to syslog.php itself, more to the config polling code, that uses "." as a separator for the hierarchy of config elements. So the hostname FQDN is split into different sub-parameters.

**Comment by kkrumm1** [comment]
I m closing this for now. Feel free to reopen.

**Pull request closed without merging**